spec/rails_helper.rb

```diff
@@ -75,3 +78,43 @@
     with.library :rails
   end
 end
+
+VCR.configure do |c|
```

Comment:
add support for VCRs for recording real interactions

Reply:
new to VCR, interested to learn more!